academy/ibc-dev/ibcmw-integrate.md

```diff
@@ -0,0 +1,156 @@
+# Integrating IBC middleware into a chain
+
+<HighlightBox type="learning">
+
+Learn how to integrate IBC middleware(s) with a base application to your chain. The following document only applies for Cosmos SDK chains.
+
+</HighlightBox>
+
+We've now seen how to develop an IBC middleware, the final step in order to use it is to integrate it into the chain. We can distinguish between a single piece of middleware or a stack of middleware and between middleware that is stateful or stateless.
+
+To integrate our middleware (stack) we must do the following in `app.go`:
+
+- If the middleware is maintaining its own state and/or processing SDK messages, then it should create and register its SDK module **only once** with the module manager
```

Comment:
Note: This is the case where defining the Middleware Module explicitly is required